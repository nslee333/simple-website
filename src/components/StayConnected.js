import Newsletter from "./Newsletter";
import Share from "./Share";

export default function Connected() {
  return (
    <>
      <div className="flex justify-evenly xs:flex-wrap">
        <Newsletter />
        <Share />
      </div>
    </>
  );
}
