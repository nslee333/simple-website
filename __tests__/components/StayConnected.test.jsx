import { render } from "@testing-library/react";
import StayConnected from "../../src/components/StayConnected";
import renderer from "react-test-renderer";

test("StayConnected renders without crashing", () => {
  render(<StayConnected />);
});

test("StayConnected renders correctly according to snapshot", () => {
  const tree = renderer.create(<StayConnected />).toJSON();

  expect(tree).toMatchSnapshot();
});
