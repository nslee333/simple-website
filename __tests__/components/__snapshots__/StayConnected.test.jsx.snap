// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StayConnected renders correctly according to snapshot 1`] = `
<div
  className="flex justify-evenly xs:flex-wrap"
>
  <div
    className="m-5 h-auto w-[42rem] min-w-[18.85rem] rounded-md bg-white p-5 sm:h-[25rem]"
  >
    <div>
      <h3
        className="p-2 text-3xl italic"
      >
        Newsletter
      </h3>
      <h4
        className="p-2 text-2xl italic"
      >
        Stay up to date
      </h4>
      <div
        className="p-2 italic"
      >
        Sign up for our monthly newsletter where we share local events in Bend, tech news and our community member spotlight!
      </div>
    </div>
    <div
      className="mx-2 my-2 flex"
    >
      <div
        className="grid  w-[100%] rounded-md bg-neutral-300 p-2"
      >
        <div
          className="flex-wrap"
          data-netlify="true"
          method="post"
          name="newsletter"
        >
          <input
            className="m-2 block rounded-md p-2"
            name="form-name"
            type="hidden"
            value="newsletter"
          />
          <label>
            <div
              hidden={true}
            >
              name
            </div>
            <input
              className="m-2 block w-[95%] min-w-[13.25rem] rounded-md p-2"
              name="name"
              placeholder="Full Name"
              required={true}
              type="text"
            />
          </label>
          <label>
            <div
              hidden={true}
            >
              email
            </div>
            <input
              className="m-2 block w-[95%] min-w-[13.25rem] rounded-md p-2"
              name="email"
              placeholder="Email"
              required={true}
              type="email"
            />
          </label>
          <button
            className="shadow-grey  mb-2  ml-2  mt-4 block  h-[2rem]  w-[20%]  min-w-[7rem]  rounded-md  bg-white  text-xl  shadow  hover:origin-bottom  hover:translate-y-0.5 hover:shadow-none"
            type="submit"
          >
            Sign up
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="m-5  h-auto  w-[42rem]  min-w-[18.85rem]  rounded-md  bg-white  p-5  sm:h-[25rem]"
  >
    <h3
      className="text-3xl italic xs:mt-2"
    >
      Have something to share?
    </h3>
    <h4
      className="text-2xl italic xs:mt-10"
    >
      We're always looking for speakers
    </h4>
    <div
      className="italic xs:mt-10"
    >
      If you have something to share, big or small, big or small, please get in touch!
    </div>
    <div
      className="mb-2 mt-10 sm:mt-32 hover:translate-y-0.5"
    >
      <a
        className="shadow-grey  rounded-lg  bg-[#7FE7D7]  p-3  text-2xl  shadow  hover:origin-bottom  hover:shadow-none"
        href="#contact"
        htmlFor="button"
      >
        Contact Us
      </a>
    </div>
  </div>
</div>
`;
